src/uikit.cs

```diff
@@ -14301,10 +14884,16 @@ interface UIViewController : NSCoding, UIAppearanceContainer, UIContentContainer
 		[Export ("shouldPerformSegueWithIdentifier:sender:")]
 		bool ShouldPerformSegue (string segueIdentifier, NSObject sender);
 
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'CanPerformUnwind2' instead.")]
+		[Deprecated (PlatformName.TvOS, 13, 0, message: "Use 'CanPerformUnwind2' instead.")]
 		[iOS (6,0)]
 		[Export ("canPerformUnwindSegueAction:fromViewController:withSender:")]
 		bool CanPerformUnwind (Selector segueAction, UIViewController fromViewController, NSObject sender);
 
+		[TV (13,0), iOS (13,0)]
+		[Export ("canPerformUnwindSegueAction:fromViewController:sender:")]
+		bool CanPerformUnwind2 (Selector segueAction, UIViewController fromViewController, [NullAllowed] NSObject sender);
```

Comment:
So they added a new function that ends up with the same signature for us 😒 

The `2` suffix makes it sound like we did something wrong... but no other good option comes to mind either. Maybe `CanPerformUnwindAction`? `CanPerformSegueUnwind`? `CanPerformUnwindSegueAction`?

Also maybe add some `#if XAMCORE_4_0` spaghetti to rename the old function `CanPerformUnwindDeprecated` and the new one `CanPerformUnwind`.

Reply:
I think I like `CanPerformUnwindSegueAction` better